comp-layouts/button.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+     <meta charset="UTF-8">
+     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <script src="https://kit.fontawesome.com/c4d0bd5372.js" crossorigin="anonymous"></script>
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="/stylesheets/main.css">
+    <link rel="stylesheet" href="/stylesheets/components/button.css">
+     <title>button</title>
+</head>
+<body>
+
+     <div class="btn-hover box">
+          <button class="btn-1"> HOVER ME! </button>
+          <button class="btn-2"> AWESOME </button>
+      </div>
+   <div class=" btn-container box">
+      <button class="btn">Click Here</button>
```

Comment:
Remove the extra space before the btn-container class. Also, indent the code properly.

Reply:
ok done